spring-cloud-gcp-pubsub-stream-binder/README.adoc

```diff
@@ -0,0 +1,29 @@
+= Spring Cloud GCP Stream for Pub/Sub
```

Comment:
I am still against READMEs in the target modules. 
There is Reference Manual on the matter.

Reply:
This suggestion came from @joaoandremartins. So should I remove this file again?

Reply:
I would move it to the Reference Manual, indeed.
But let's do that after when we agree how `GcpPubSubEmulatorConfiguration` should look!

Reply:
I think people read GitHub READMEs..
At least I do?

The reason I didn't suggest adding this to the refdoc is because I wonder if the refdoc is the right place to tell people they can use the emulator to test?

Reply:
Let it be.
But don't forget to support all of them and deal with duplication! 😄 

I think that it is still good point to add such a note into the Reference Manual.
As a sample we can take into account that Spring Boot documents all the emulator and embedded services configurations in its Reference Manual.